src/helpers/draftBills.js

```diff
@@ -292,15 +293,25 @@ exports.computeBillingInfoForEvents = (events, service, fundings, billingStartDa
     }
 
     if (moment(event.startDate).isBefore(startDate)) startDate = moment(event.startDate);
+
+    if (get(matchingService, 'billingItems.length')) {
```

Comment:
Est-ce que ce if est nécessaire? Si billingItems est vide, normalement on ne rentrera pas dans la boucle for